pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -18,6 +18,7 @@ assertExecutable() {
 # --set-default VAR VAL : like --set, but only adds VAR if not already set in
 #                         the environment
 # --unset       VAR     : remove VAR from the environment
+# --chdir       DIR     : change working directory (use instead of --run "cd DIR")
```

Comment:
Should we log a warning or error when this is done?

Reply:
Anything short of parsing shell syntax would have to be an ugly heuristic like "the command starts with cd" which is bound to miss some legitimate use cases, so I vote no